src/commands/release.ts

```diff
@@ -0,0 +1,54 @@
+import { Arguments } from 'yargs';
+import * as git from 'simple-git/promise';
+import * as opn from 'opn';
+import * as readPkg from 'read-pkg';
+import * as semver from 'semver';
+import * as writePkg from 'write-pkg';
+
+const productionBranchname = 'production';
+
+const validateVersion = (version: string) =>
+  !!semver.valid(version) || false;
+
+const releaseCommand = async (
+  command: any /* ask Richard about this */,
```

Comment:
`command` is the command which is being selected. speaking of `release` in this case. what is the specific question?